azure_arc_data_jumpstart/aks/arm_template/artifacts/azuredeploy.parameters.example.json

```diff
@@ -1,42 +1,45 @@
 {
-    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
-    "contentVersion": "1.0.0.0",
-    "parameters": {
-        "sshRSAPublicKey": {
-            "value": "ssh-rsa THISISTOPSECRETSTUFF1234XXXXXXXXXXXXXtEbhbVqD bilbo@MacBook-Pro.localdomain"
-        },
-        "spnClientId": {
-            "value": "33333333-XXXX-YYYY-XXXX-YTYTYTYT"
-        },
-        "spnClientSecret": {
-            "value": "33333333-XXXX-YTYT-9c21-7777777777"
-        },
-        "spnTenantId": {
-            "value": "33333333-XXXX-41af-1111-7777777777"
-        },
-        "windowsAdminUsername": {
-            "value": "arcdemo"
-        },
-        "windowsAdminPassword": {
-            "value": "ArcPassword123!!"
-        },
-        "myIpAddress": {
-            "value": "88.88.88.88"
-        },
-        "logAnalyticsWorkspaceName": {
-            "value": "Arc-Data-LA"
-        },
-        "deploySQLMI": {
-            "value": false
-        },
-        "deployPostgreSQL": {
-            "value": false
-        },
-        "kubernetesVersion": {
-            "value": "1.18.17"
-        },
-        "dnsPrefix": {
-            "value": "arcdata"
-        }
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "sshRSAPublicKey": {
+      "value": "ssh-rsa THISISTOPSECRETSTUFF1234XXXXXXXXXXXXXtEbhbVqD bilbo@MacBook-Pro.localdomain"
+    },
+    "spnClientId": {
+      "value": "33333333-XXXX-YYYY-XXXX-YTYTYTYT"
+    },
+    "spnClientSecret": {
+      "value": "33333333-XXXX-YTYT-9c21-7777777777"
+    },
+    "spnTenantId": {
+      "value": "33333333-XXXX-41af-1111-7777777777"
+    },
+    "windowsAdminUsername": {
+      "value": "arcdemo"
+    },
+    "windowsAdminPassword": {
+      "value": "ArcPassword123!!"
+    },
+    "myIpAddress": {
+      "value": "88.88.88.88"
+    },
+    "logAnalyticsWorkspaceName": {
+      "value": "Arc-Data-LA"
+    },
+    "deploySQLMI": {
+      "value": false
+    },
+    "deployPostgreSQL": {
+      "value": false
+    },
+    "templateBaseUrl": {
+      "value": "https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_arc_data_jumpstart/aks/arm_template/"
+    },
+    "kubernetesVersion": {
+      "value": "1.18.17"
```

Comment:
```suggestion
      "value": "1.19.11"
```

Reply:
@likamrat I will test the deployment with the new AKS version once more.